### ISSUE TITLE
android-intents/v1.3.0 build

### ISSUE THREAD

**Issue by alejandro-pnz** [jitpack/jitpack.io#748]
Log https://jitpack.io/com/github/ifunny/android-intents/v1.3.0/build.log

During build on JitPack server, gradle give the next errors:

```
ERROR: Gradle wrapper not found. Please add. Using default gradle to build.
Minimum supported Gradle version is 2.14.1.  Current version is 2.7. If using the gradle wrapper, try editing the distributionUrl in /home/jitpack/build/gradle/wrapper/gradle-wrapper.properties to gradle-2.14.1-all.zip
```

I've added distribution url into the gradle/wrapper/gradle-wrapper.properties, but this has no effect. 
What I need to do?

**Comment by alejandro-pnz** [comment]
Any possible solutions?

**Comment by jitpack-io** [comment]
Hi,

Looks like 1.4.0 was successful.

**Comment by alejandro-pnz** [comment]
Hello, we've solved the problem in 1.4.0 release. The problem was in the missing gradlew file.
